src/en/information/newsletter/index.md

```diff
@@ -6,8 +6,8 @@ title: Subscribe to our newsletter
 Every now and then, we send a newsletter with news related to our association and front-end development. Don't want to miss out? Subscribe here.
 
 <form action="https://fronteers.us10.list-manage.com/subscribe/post?u=d8d859a3bce138807ce785741&amp;id=49b991ad18" method="post" class="generated">
-    <p><label for="subscribe-email">Email address <em>required</em></label> <input type="email" name="EMAIL" id="subscribe-email" class="text"></p>
-    <p><label for="subscribe-voornaam">First name</label> <input type="text" name="FNAME" id="subscribe-voornaam" class="text"></p>
-    <p><label for="subscribe-achternaam">Last name</label> <input type="text" name="LNAME" id="subscribe-achternaam" class="text"></p>
+    <p><label for="subscribe-email">Email address <em>required</em></label> <input type="email" name="EMAIL" id="subscribe-email" class="text" autocomplete="email"></p>
+    <p><label for="subscribe-voornaam">First name</label> <input type="text" name="FNAME" id="subscribe-voornaam" class="text" autocomplete="given-name"></p>
+    <p><label for="subscribe-achternaam">Last name</label> <input type="text" name="LNAME" id="subscribe-achternaam" class="text" autocomplete="family-name"></p>
```

Comment:
We do a lovely job collecting the name in the other forms instead of the westernised first/last or given/family name. See: https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

If possible, I would *strongly* suggest to collect a singular name.